tasks/apt.yml

```diff
@@ -41,6 +41,12 @@
     - python3-dev
     - python-virtualenv
 
+- name: Install python3-pyparsing
+  when: ansible_distribution == "Debian"
```

Comment:
isn't this missing a condition to specify that it should be executed only on debian 9?